www/notifications/inner.js

```diff
@@ -143,8 +143,9 @@ define([
             $(loadmore).click();
         }
 
-        common.mailbox.subscribe(["notifications"], {
+        common.mailbox.subscribe(["notifications", "reminders"], {
             onMessage: function (data, el) {
+                el.innerHTML = el.innerHTML.replace('<i', '<span class="cp-avatar"><span class="cp-avatar-default" alt="" aria-hidden="true"><i').replace('/i>', '/i></span></span>');
```

Comment:
Maybe better to use some DOM manipulation + hyperscript here to have a more robust/readable solution than `sed`ing the `innerHTML` directly?